apps/api/src/app/health/health.controller.ts

```diff
@@ -3,17 +3,23 @@ import { ApiExcludeController } from '@nestjs/swagger';
 import { HealthCheck, HealthCheckService } from '@nestjs/terminus';
 import { HealthIndicatorFunction } from '@nestjs/terminus/dist/health-indicator';
 import { DalService } from '@novu/dal';
+import {
+  CacheServiceHealthIndicator,
+  InMemoryProviderServiceHealthIndicator,
+  TriggerQueueServiceHealthIndicator,
+} from '@novu/application-generic';
 
 import { version } from '../../../package.json';
-import { CacheServiceHealthIndicator } from './cache-health-indicator';
 
 @Controller('health-check')
 @ApiExcludeController()
 export class HealthController {
   constructor(
     private healthCheckService: HealthCheckService,
     private dalService: DalService,
-    private cacheHealthIndicator: CacheServiceHealthIndicator
+    private cacheHealthIndicator: CacheServiceHealthIndicator,
+    private inMemoryHealthIndicator: InMemoryProviderServiceHealthIndicator,
+    private triggerQueueHealthIndicator: TriggerQueueServiceHealthIndicator
```

Comment:
First steps to be able to complete in a different PR the checks for the graceful bootstrap.